IntegrationTests/common/script/CompareMemPrintsFW.py

```diff
@@ -233,7 +341,7 @@ def comparePredefined(args):
     parser.add_argument("-c","--comparison_filename",default="output.txt",help="The filename of the testbench output file (default = %(default)s)")
     parser.add_argument("-f","--fail_on_error",default=False,action="store_true",help="Raise an exception on the first error as opposed to simply printing a message (default = %(default)s)")
     parser.add_argument("-l","--file_location",default="./",help="Location of the input files (default = %(default)s)")
-    parser.add_argument("-p","--predefined",default=False,action="store_true",help="Run predefined comparisons (default = %(default)s)")
+    parser.add_argument("-p","--predefined",default="",help="Run predefined comparisons. Options: PRMEMC, IRVMR, VMR (default = %(default)s)")
```

Comment:
Could script get default value of option "-p" from:
import os
l1TrkChain = os.getcwd().split('/')[-2]

Reply:
Not if you want to be able to run it without any predefined option? But this is irrelevant now when I have automated the predefined comparison.